docs/settingUp.md

```diff
@@ -21,7 +21,7 @@ The full tool stack is given at the [end of this document](#tool-stack).
 ## Prerequisites
 Important: When a version is specified, please install that version instead of the latest version available.
 
-1. Install GitHub for Windows/Mac (recommended), or at least, Git.
+1. Install Source Tree or other similar Git Client, or at least Git.
```

Comment:
Make it `Install SourceTree (recommended) or ...`. The "recommended" part needs to be retained, and SourceTree's trademarked name is with no space between the two words.